src/RunProcessAsTask.Tests/ProcessExTests.cs

```diff
@@ -60,13 +60,16 @@ public void WhenProcessRunsNormally_ReturnsExpectedResults()
                 Assert.True(results.RunTime.TotalMilliseconds >= millisecondsToSleep);
             }
 
-            [Fact]
-            public void RunLotsOfOutputForFiveMinutes()
+            [Theory]
+            [InlineData(10)]
+            //[InlineData(60)]
+            //[InlineData(300)]

```

Comment:
No reason for multiple values here; keep the old version of one plain fact.

Reply:
I would suggest running that 5min test locally. That's too long for a single CI test

Reply:
whatever value to pick: pick one; if you need to edit the source anyhow, then doing it via the theory infrastructure is merely more indirect; clearer to edit the time expression directly.